2025-05-19 20:17:44.505 [com.example.App.main()] INFO  com.example.S3ToDynamoUploader - DynamoDB表已存在: s3_exercise
2025-05-19 20:17:44.509 [com.example.App.main()] INFO  com.example.App - [DynamoDB] 表创建/验证完成: s3_exercise
2025-05-19 20:17:44.885 [com.example.App.main()] ERROR com.example.App - [System] 程序执行失败: java.io.FileNotFoundException: Could not locate Hadoop executable: E:\hadoop-3.4.1\bin\winutils.exe -see https://wiki.apache.org/hadoop/WindowsProblems
